SimCalorimetry/HGCalSimProducers/python/hgcHitAssociation_cfi.py

```diff
@@ -0,0 +1,3 @@
+from SimCalorimetry.HGCalAssociatorProducers.layerClusterAssociatorByEnergyScore_cfi import layerClusterAssociatorByEnergyScore as LCAssocByEnergyScoreProducer
+
+from RecoLocalCalo.HGCalRecProducers.hgcalRecHitMapProducer_cfi import hgcalRecHitMapProducer as HGCRecHitMapProducer
```

Comment:
this is a bit outside of reco, but still: the module instance names are expected to be start with lower case (same as the default module labels), unless there are some good reasons